### PR TITLE
fix handle ID and Fsync

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

File: loopback.go [hunk 1]
```diff
@@ -311,6 +289,27 @@ func fuseOpenFlagsToOSFlagsAndPerms(
...
+	n.flushers[h] = true
+}
+
+func (n *Node) forgetHandle(h *Handle) {
+	n.lock.Lock()
+	defer n.lock.Unlock()
+	if n.flushers == nil {
+		return
+	}
+	if !n.flushers[h] {
```
Comment (strib) [inline loopback.go]: It's safe to skip this check and just do the `delete` below.

File: loopback.go [hunk 2]
```diff
@@ -360,9 +362,15 @@ func (n *Node) Create(
...
 
-	node := &Node{realPath: filepath.Join(n.realPath, req.Name),
-		isDir: req.Mode.IsDir(), fs: n.fs}
-	resp.Handle = n.fs.newHandle(h)
+	node := &Node{
+		realPath: filepath.Join(n.realPath, req.Name),
+		isDir:    req.Mode.IsDir(),
+		fs:       n.fs,
+	}
+	node.rememberHandle(h)
```
Comment (strib) [inline loopback.go:367]: No need to set `resp.Handle` anymore?

**Review by strib — APPROVED** [review]

File: loopback.go (hunk 1 above)
Comment (songgao) [reply loopback.go]: Ah nice; thx!

File: loopback.go (hunk 2 above)
Comment (songgao) [reply loopback.go:367]: Correct; `resp.Handle` is a HandleID, which gets overridden by basil anyway, so there's no need to generate them here

**Pull request merged**